clang/lib/Parse/ParseExpr.cpp

```diff
@@ -3999,6 +4008,23 @@ std::optional<AvailabilitySpec> Parser::ParseAvailabilitySpec() {
       Actions.CodeCompletion().CodeCompleteAvailabilityPlatformName();
       return std::nullopt;
     }
+
+    if (Tok.is(tok::identifier) && GetLookAheadToken(1).is(tok::equal)) {
```

Comment:
suggest: `tok::colon` instead of `tok:equal`, so it matches the `feat: <feature>` style ones